Orso.Arpa.Persistence/Seed/Translations/Localization/de.json

```diff
@@ -1,7 +1,7 @@
 [
   {
-    "Key": "This request requires a valid JWT access token to be provided",
```

Comment:
the files in the Localization folder are generated automatically during migration and must not be updated manually (see readme file in this folder)